<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Confirm_order</class>
 <widget class="QWidget" name="Confirm_order">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>606</width>
    <height>546</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>Franklin Gothic Book</family>
    <pointsize>12</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="label_allConfirmed">
     <property name="font">
      <font>
       <family>Franklin Gothic Book</family>
       <pointsize>16</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Все заказы подтверждены</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QPushButton" name="pushButton_confirmation">
     <property name="text">
      <string>Подтвердить заказ</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QPushButton" name="pushButton_backToAdmin">
     <property name="text">
      <string>Вернуться на главную</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QTableWidget" name="tableWidget_confirmOrders"/>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
